modules/data-loading/partials/load-part2-create-data-source.adoc

```diff
@@ -24,7 +24,7 @@ key: <value>
 String literals can be enclosed with a double quote `"`, triple double quotes `"""`, or triple single quotes `'''`.
 Double quotes `"` in the JSON can be omitted if the key name does not contain a colon `:` or comma `,`.
 
-[source,php]
+[source,gsql]
 .Alternate quote syntax for inline JSON data
 CREATE DATA_SOURCE s1 = """{
 "type": "<type>",
```

Comment:
line 34

Both a period `.` and `_` can be used for separation in the specified key name. Example: `first.second` or `first_second`.